app-modules/core/src/Http/Controllers/Api/PasswordController.php

```
@@ -0,0 +1,84 @@
+            'email' => ['required', 'email'],
+        ]);
+
+        $status = Password::broker()->sendResetLink($validated);
+
+        if ($status === Password::RESET_LINK_SENT) {
+            return response()->json(['message' => __($status)]);
+        }
+
+        throw ValidationException::withMessages([
+            'email' => [__($status)],
+        ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Validation\ValidationException' in method 'forgot'. (undefined)

(StaticAccess)